grammar/grammar.y

```diff
@@ -2,16 +2,18 @@ Root <- skip ContainerMembers eof
 
 # *** Top level ***
 ContainerMembers
-    <- TestDecl ContainerMembers
-     / TopLevelComptime ContainerMembers
-     / KEYWORD_pub? TopLevelDecl ContainerMembers
-     / ContainerField COMMA ContainerMembers
-     / ContainerField
-     /
+    <- container_doc_comment? (
+         TestDecl ContainerMembers
+         / TopLevelComptime ContainerMembers
+         / doc_comment? KEYWORD_pub? TopLevelDecl ContainerMembers
+         / ContainerField COMMA ContainerMembers
+         / ContainerField
+         /
```

Comment:
Ah, this is why I can have a doc comment at the end of a struct.  It matches the current implementation though, so good change.